src/configuration.ts

```diff
@@ -436,7 +436,7 @@ export const baseDefaults: IBaseConfiguration = {
   rootPath: '${workspaceFolder}',
   // keep in sync with sourceMapPathOverrides in package.json
   sourceMapPathOverrides: {
-    'webpack:///*': '*',
+    'webpack://?:*/*': '*',
```

Comment:
So this basically ignores the first segment and matches with the second segment? This was added for projects that end up with `webpack:///absolute/path` and it sounds like we can't handle both that and this other syntax without more smarts

Reply:
Oh, it goes from 3 / to 2, and I read the description more closely, it sounds like you are on top of it 👍

Reply:
Yep 😄